tests/functional/functional-tests.js

```diff
@@ -2507,7 +2501,7 @@ describe('functional tests', function () {
           if (err) {
             return done(err)
           }
-          if (isArray(tagList)) {
+          if (Array.isArray(tagList)) {
```

Comment:
a request to update line number  **3775** to
```suggestion
              const stsEndPoint = 'http://localhost:9000'
              const stsEndPoint = 'http://' + clientConfigParams.endPoint + ':' + clientConfigParams.port

```